include/fc/config.hpp

```diff
@@ -6,4 +6,9 @@
 #ifndef FC_MAX_LOG_OBJECT_DEPTH
    // how many levels of nested objects are displayed in log messages
    #define FC_MAX_LOG_OBJECT_DEPTH 200
-#endif
\ No newline at end of file
+#endif
+
+#ifndef FC_MAX_PREALLOC_SIZE
+// how many elements will be reserve()d when deserializing vectors
+#define FC_MAX_PREALLOC_SIZE (200UL)
```

Comment:
Better change the default value to `2^x` to get better performance since `*2` is used in loop.